sdk/src/main/java/com/mapbox/maps/MapInitOptions.kt

```diff
@@ -51,22 +51,25 @@ data class MapInitOptions constructor(
       MapOptions.Builder().applyDefaultParams(context).build()
 
     /**
-     * Get a default selection of Mapbox created plugins.
+     * Default map view plugin registry.
+     *
+     * If creating [MapView] from xml -
+     * all plugins listed here will be applied to given [MapView] on startup.
+     *
+     * By default all Mapbox plugins are present in this list.
      */
-    fun getDefaultPlugins(): List<String> {
-      return listOf(
-        PLUGIN_CAMERA_ANIMATIONS_CLASS_NAME,
-        PLUGIN_COMPASS_CLASS_NAME,
-        PLUGIN_LOGO_CLASS_NAME,
-        PLUGIN_GESTURE_CLASS_NAME,
-        PLUGIN_ATTRIBUTION_CLASS_NAME,
-        PLUGIN_LOCATION_COMPONENT_CLASS_NAME,
-        PLUGIN_SCALE_BAR_CLASS_NAME,
-        PLUGIN_MAPOVERLAY_CLASS_NAME,
-        PLUGIN_ANNOTATION_CLASS_NAME,
-        PLUGIN_LIFE_CYCLE_CLASS_NAME
-      )
-    }
+    val defaultPluginList = mutableListOf(
```

Comment:
nit: should this be defined int the Plugin.kt ?

Also thinking about multiple MapView use cases, if user passes the Custom plugin with a plugin instance, it seems doesn't work well, as the plugin depends on the state of the MapView.

Reply:
Made it immutable.
Thus it does not matter much where it is located but let's leave it here.